src/v3/src/components/CaptchaContainer/CaptchaContainer.test.ts

```diff
@@ -31,14 +53,40 @@ jest.mock('../../contexts', () => ({
         },
       },
     },
-    widgetProps: {},
+    widgetProps: {
+      baseUrl: 'https://test.okta.com',
+    },
     recaptchaOptions: undefined,
   }),
 }));
 
+jest.mock('../../util', () => ({
+  getBaseUrl: () => 'https://test.okta.com',
```

Comment:
This mock shadows a real read to the `widgetProps` on new L56-L58 I think. One or the other is not needed?